src/components/message/MessageStyles.tsx

```diff
@@ -0,0 +1,143 @@
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import styled from '@emotion/styled';
+import { IconCheckCircle, IconError, IconInfo, IconQuestionMark, IconWarning } from '../icons';
+import { MessageInlineSize, MessageType } from './MessageTypes';
+
+export const MESSAGE_TYPE_ICON = {
+  [MessageType.Info]: <IconInfo color="echoes-color-icon-info" />,
+  [MessageType.Danger]: <IconError color="echoes-color-icon-danger" />,
+  [MessageType.Warning]: <IconWarning color="echoes-color-icon-warning" />,
+  [MessageType.Success]: <IconCheckCircle color="echoes-color-icon-success" />,
+  [MessageType.Discover]: <IconQuestionMark color="echoes-color-icon-accent" />,
+};
+
+export const MESSAGE_CALLOUT_TYPE_STYLE = {
+  [MessageType.Info]: {
+    '--message-background': 'var(--echoes-color-background-info-weak)',
+    '--message-title-color': 'var(--echoes-color-text-info-bold)',
+  },
+  [MessageType.Danger]: {
+    '--message-background': 'var(--echoes-color-background-danger-weak-default)',
+    '--message-title-color': 'var(--echoes-color-text-danger-bold)',
+  },
+  [MessageType.Warning]: {
+    '--message-background': 'var(--echoes-color-background-warning-weak)',
+    '--message-title-color': 'var(--echoes-color-text-warning-bold)',
+  },
+  [MessageType.Success]: {
+    '--message-background': 'var(--echoes-color-background-success-weak)',
+    '--message-title-color': 'var(--echoes-color-text-success-bold)',
+  },
+  [MessageType.Discover]: {
+    '--message-background': 'var(--echoes-color-background-accent-weak-default)',
+    '--message-title-color': 'var(--echoes-color-text-accent-bold)',
+  },
+};
+
+export const MESSAGE_INLINE_TYPE_STYLE = {
+  [MessageType.Info]: {
+    '--message-text-color': 'var(--echoes-color-text-info)',
+  },
+  [MessageType.Danger]: {
+    '--message-text-color': 'var(--echoes-color-text-danger)',
+  },
+  [MessageType.Warning]: {
+    '--message-text-color': 'var(--echoes-color-text-warning)',
+  },
+  [MessageType.Success]: {
+    '--message-text-color': 'var(--echoes-color-text-success)',
+  },
+  [MessageType.Discover]: {
+    '--message-text-color': 'var(--echoes-color-text-accent)',
+  },
+};
+
+export const MessageCalloutContainer = styled.div`
```

Comment:
Let's add the `displayName` to all these components, it makes it cleaner to debug in the compiled app